src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -183,6 +204,9 @@ class Mangago : ParsedHttpSource() {
             dateFormat.parse(element.select("td:last-child").text().trim())?.time
         }.getOrNull() ?: 0L
         scanlator = element.selectFirst("td.no a, td.uk-table-shrink a")?.text()?.trim()
+        if (scanlator.isNullOrBlank()) {
+            scanlator = "Unknown"
```

Comment:
Since `scanlator` can be null, why not default to that (or not assigning anything to it at all)? So something like:
```kt
element.selectFirst("td.no a, td.uk-table-shrink a")?.text()?.also { scanlator = it }

```

Reply:
but then there won't be any scanlator in it right? at least when I tested it here, no scanlator appears at all. It makes it impossible to use the scanlator filters